.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
       rev: 23.11.0
       hooks:
           - id: black
-            args: [--safe, --quiet, --target-version, py36]
+            args: [--safe, --quiet, --target-version, py37]
```

Comment:
Please could you also switch to use `https://github.com/psf/black-pre-commit-mirror` for the repo? It's a bit faster.

And we might as well update them too: `pre-commit autoupdate --jobs 0`

Reply:
Well Black di some significant changes so I may suggest doing it in a separate PR #252

Reply:
I don't see any changes from Black in https://github.com/pytest-dev/pytest-rerunfailures/pull/252 :)